src/Slice.cpp

```diff
@@ -20,7 +21,7 @@ Slice::Slice(const size_t num_mesh_groups)
 
 void Slice::compute()
 {
-    spdlog::info("All settings: {}", scene.getAllSettingsString());
+    fmt::print("All settings: {}", scene.getAllSettingsString()); // Print to console do not log, otherwise Sentry won't upload the crash because it is too big.
```

Comment:
As the sentry spdlog sink has been removed, you can revert it back to using spdlog

Reply:
good point, feel free to change